vnext/ReactUWP/Modules/NativeUIManager.cpp

```diff
@@ -26,8 +26,18 @@ using namespace xaml::Media;
 namespace react {
 namespace uwp {
 
-static YogaNodePtr make_yoga_node() {
-  YogaNodePtr result(YGNodeNew());
+YGConfigRef getYogaConfig(bool useLegacyStretchBehaviour) {
+  static YGConfigRef yogaConfig;
+  static std::once_flag onceFlag;
+  std::call_once(onceFlag, [useLegacyStretchBehaviour]() {
```

Comment:
>std::call_once( [](start = 2, length = 15)

Since we store result in a static variable, we can avoid using this function.
It can be done by using a lambda that us called that is called in-place:
static YGConfigRef yogaConfig = [useLegacyStretchBehaviour]() {
    auto yogaConfig = YGConfigNew();
    YGConfigSetUseLegacyStretchBehaviour(yogaConfig, useLegacyStretchBehaviour);
     return yogaConfig;
  }();

Note the '()' in the end to call the lambda.

Reply:
I actually moved it out of the static, and made it a member of the UIManager, since I want to allow multiple instances to have different settings.  That way multiple instances can migrate separately over time in complex apps.